src/main.rs

```diff
@@ -1,10 +1,16 @@
 use anyhow::Result;
-use cargo_generate::{generate, Cli};
+use cargo_generate::{app_config_path, generate, list_favorites, AppConfig, Cli};
 use clap::Parser;
 
 fn main() -> Result<()> {
     let Cli::Generate(args) = Cli::parse();
-    generate(args)?;
+    let app_config: AppConfig = app_config_path(&args.config)?.as_path().try_into()?;
```

Comment:
This is a nice little refactoring if you ask me...
Makes it more clearcut between the `list-favorites` and normal `generate` functionality..